hw/ip/otbn/dv/otbnsim/standalone.py

```diff
@@ -43,6 +43,9 @@ def main() -> int:
 
     sim = StandaloneSim()
     exp_end_addr = load_elf(sim, args.elf)
+    key0 = int((str("deadbeef") * 12), 16)
+    key1 = int((str("badf00d") * 12), 16)
```

Comment:
Nit: You don't actually need the string cast. So this could just be `key0 = int('deadbeef' * 12, 16)`. A very clever trick though!